fiftyone/utils/flash.py

```diff
@@ -96,6 +98,7 @@ def compute_flash_embeddings(
     embeddings_field=None,
     batch_size=None,
     num_workers=None,
+    transform_kwargs=None,
```

Comment:
Although it is a small breaking change, I think updating `trainer_kwargs` to be a dict (defaulting to `trainer_kwargs=None`) rather than `**trainer_kwargs` would make sense, for consistency with `transform_kwargs` here.

Make this change in analogous methods like `apply_flash_model()` too.

With the above changes in-place, you could then rename the `**trainer_kwargs` arguments of `SampleCollection.apply_model()` and `SampleCollection.compute_embeddings()` to just `**kwargs`, and update the docs to just explain that these are additional model-specific keyword arguments passed through to the underlying inference implementation (after all, users may want to pass `transform_kwargs` now). This would be an improvement because its bad to have Flash-specific things in the docstring of `SampleCollection` methods; the latter should be generic.